plugin/evm/customheader/base_fee.go

```diff
@@ -58,5 +60,7 @@ func EstimateNextBaseFee(
 	}
 
 	timestamp = max(timestamp, parent.Time, *config.ApricotPhase3BlockTimestamp)
-	return BaseFee(config, parent, timestamp)
+	// Use a fake header with this timestamp.
+	header := &types.Header{Time: timestamp}
+	return BaseFee(config, parent, header)
```

Comment:
this is probably much worse than all these complicated stuff with handling ms and sec. as `BaseFee` can unintentionally require stuff from header in the future.

Reply:
and I guess it would not even work since we don't set `ms` timestamp in the here.